content/linux-gaming.md

```diff
@@ -28,13 +29,13 @@ Open the <u>Pop!_Shop</u> application then either search for Steam or by clickin
 
 ![Pop!_Shop Steam](/images/linux-gaming/pop-shop_steam.png)
 
-Once installed, use the Activities Overview to search for and run <u>Steam</u>.
+Once installed, use the Application button to search for and run <u>Steam</u>.
```

Comment:
The button says "Applications", so I'm thinking Application should probably be plural to match the button, right?